apps/spruce/src/components/PatchesPage/RequesterSelector.tsx

```diff
@@ -0,0 +1,47 @@
+import { Combobox, ComboboxOption } from "@leafygreen-ui/combobox";
+import { useStatusesFilter } from "hooks";
+import { PatchPageQueryParams } from "types/patch";
+
+export const RequesterSelector: React.FC = () => {
+  const { inputValue: statusVal, setAndSubmitInputValue: statusValOnChange } =
+    useStatusesFilter({ urlParam: PatchPageQueryParams.Requesters });
+
+  return (
+    <Combobox
+      label=""
+      data-cy="requester-selector"
+      placeholder="Select patch types"
+      value={statusVal}
+      multiselect
+      onChange={statusValOnChange}
+      overflow="scroll-x"
+    >
+      {options.map(({ displayName, key, value }) => (
+        <ComboboxOption
+          key={key}
+          value={value}
+          displayName={displayName}
+          data-cy={`${value}-option`}
+        />
+      ))}
+    </Combobox>
+  );
+};
+
+enum Requesters {
+  GithubPRRequester = "github_pull_request",
+  PatchVersionRequester = "patch_request",
+}
```

Comment:
I think you can instead use `requesterSubscriberOptions` [here](https://github.com/evergreen-ci/ui/blob/38424427727d73834fe5159edc82a4088c248d08/apps/spruce/src/constants/triggers.ts#L56-L62) (imo "Patch" may also be clearer to users since it's terminology used elsewhere)